feat: mention local modules (#3877)
facebook/react-native-website: docs/native-modules-intro.md

## Native Module Setup

> There are three ways to write a native module for your React Native application:

Reviewer: ```suggestion
There are different ways to write a native module for your React Native application:
```

so if in the future we add/remove one of the way, we don't have to change this line anymore.

> 1. Creating local library that can be imported in your React Native application and can be created with one command. Read [Creating local libraries](local-library-setup) guide to learn more.

Reviewer: ```suggestion
1. Creating a local library that can be imported in your React Native application. Read [Creating local libraries](local-library-setup) guide to learn more.
```
